test/playback.test.js

```diff
@@ -23,9 +14,22 @@ const playFor = function(player, time, cb) {
         return checkPlayerTime();
       }
       cb();
-    }, 10);
+    }, 100);
   };
 
+  if (player.paused()) {
+    const playPromise = player.play();
+
+    // Catch/silence error when a pause interrupts a play request
+    // on browsers which return a promise
```

Comment:
Might be worth adding in the comment above here (and removing this, since we no longer silence the error)